doc-content/enterprise-only/openshift/githooks-proc.adoc

```diff
@@ -0,0 +1,80 @@
+[id='githooks-proc']
+= Providing the Git hooks directory
+
+If you configure the `GIT_HOOKS_DIR` parameter, you must provide a directory of Git hooks. You must mount this directory on the {CENTRAL} deployment. 
+
+The typical use of Git hooks is interaction with an upstream repository. To enable Git hooks to push commits into an upstream repository, you must also provide a secret key that corresponds to a public key configured on the upstream repository.
```

Comment:
Try to avoid the passive, so possible: Typically, Git hooks are used to interact with (or for interaction with)...

Reply:
Any prereqs?